love-letter-app/src/app/ui/animatedCard.tsx

```diff
@@ -1,20 +1,28 @@
 "use client";
-import Image from "next/image";
+import { useState } from "react";
+import FrontSVG from "./frontsvg";
+import BackSVG from "./backsvg";
+import "@/app/flip.css";
 
-export default function AnimatedCard(props: { svg: string | undefined }) {
+export default function AnimatedCard(props: {
+  svg: string | undefined;
```

Comment:
look into using a question mark for defining a type that can also be undefined.
For example: 
svg?: string